Wireframe/index.html

```diff
@@ -3,27 +3,76 @@
+         Together, these elements form the foundation of an organized and maintainable software development project
+       
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src=https://miro.medium.com/v2/resize:fit:1200/1*oppPsIqwEiB_kndtd6GWxA.png alt="" />
+        <h2>README file</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          
         </p>
         <a href="">Read more</a>
       </article>
-    </main>
+      <section class="mini-articles">
+      <article>
+     <img src="https://www.archimetric.com/wp-content/uploads/2022/02/01-youtube-wireframe-example.png"
+       <div aria-colcount="article-info">
+        <h3>wireframe </h3>
+        <p>A wireframe is a simple visual representation of a webpage, app, or software interface, outlining
+          its structure and layout without detailed design elements. It serves as a blueprint, showing where
+          key components like buttons, images, and text will be placed.
+          Wireframes are usually created in black-and-white or grayscale and focus on functionality rather
+          than aesthetics. They help designers, developers, and stakeholders plan the user experience (UX)
+          before moving on to full design and development.
+          Common tools for creating wireframes include Figma, Adobe XD, Balsamiq, and Sketch
+        </p> 
+          <a href="">Read more</a>
+      </div>
+   </article>
+  </section>
+    <section>
+    <article>
+      <img src="https://miro.medium.com/v2/resize:fit:1400/1*mtsk3fQ_BRemFidhkel3dA.png">
+       <div aria-colcount="article-info">
+        <h3> Git </h3>
+        <p>
+          Git is a version control system that helps developers track changes in their code, collaborate
+          efficiently, and manage project history. It allows multiple people to work on the same project
+          without overwriting each other’s work.
+
+          Key Features of Git:
+Version Control – Tracks changes and allows reverting to previous versions.
+Branching & Merging – Developers can work on separate branches and merge changes when ready.
+ Collaboration – Teams can contribute without conflicts using platforms like GitHub, GitLab, or Bitbucket.
+ Distributed System – Every developer has a full copy of the project history.
+
+Basic Git Commands:
+git init – Initializes a Git repository.
+git add . – Stages changes for commit.
+git commit -m "Message" – Saves changes with a message.
+git push – Uploads changes to a remote repository.
+git pull – Retrieves the latest changes from a remote repo.
+Git is widely used in software development to ensure code safety, collaboration, and efficient
+project management.
+</p>
+  <a href="">Read more</a>
+ </main>
     <footer>
       <p>
         This is the default, provided code and no changes have been made yet.
```

Comment:
You should add some styling to your footer which allows user to  distinguish where the footer starts and ends

Reply:
I can't see any changes, Could you also look my other feedbacks as well

Reply:
Still don't see any changes, are you sure you committed your recent changes to your PR ?